repository_service_tuf_api/api/bootstrap.py

```diff
@@ -28,8 +28,10 @@ def get():
     "/",
     summary="Bootstrap the system with initial signed Metadata.",
     description=(
-        "Initialize the TUF Repository with initial signed Metadata and "
-        "Settings."
+        "Submit an asynchronous task to initialize the TUF Repository with "
+        "initial signed Metadata and Settings. "
+        "Check the status and result using the task ID and the "
+        "`get task state` endpoint."
```

Comment:
`get task state` does not sound clear within the API documentation. It can be done to the other endpoints.
```suggestion
        "Use the task ID to retrieve the task status in the endpoint "
        "/api/v1/task."
```

Reply:
Hi @kairoaraujo , thank you for you response. I update the PR with your suggestions, please let me know if I need to improve or fix something ;)